openmrs/src/test/java/org/motechproject/openmrs/it/MRSTasksIntegrationBundleIT.java

```diff
@@ -210,6 +220,39 @@ public void testOpenMRSProviderDataSourceAndCreateEncounterAction() throws Inter
         checkIfProviderWasCreatedProperly();
     }
 
+    @Test
+    public void testCreatePatientPostActionParameter() throws InterruptedException {
```

Comment:
1. this task logic should be in separate method
2. please use english example names : John Smith and so on